src/components/AllArticles.jsx

```diff
@@ -7,17 +7,20 @@ import TopicsList from "./TopicsList";
 const AllArticles = ( )=> {
     const [articles, setArtcicles] = useState ([]);
     const [isLoading, setIsLoading] = useState(true)
+    const [sortBy, setSortBy] =useState('created_at')
+    const [orderBy, setOrderBy] =useState('asc')
+
     const {slug} = useParams()
 
 
     useEffect(()=> {
         setIsLoading(true)
-        getArticles(slug).then((res)=> {
+        getArticles(slug, sortBy, orderBy).then((res)=> {
```

Comment:
Awesome re-use of the getArticles function so this works for querys and missing querys 👍